### PR TITLE
Enadeau1 privacy

### PR DESCRIPTION
**Pull request by enadeau1**

### CODE REVIEW

File: sp800-63a/sec8_privacy.md [hunk 1]
```diff
@@ -32,9 +32,11 @@ An effective notice will take into account user experience design standards and
 
 ### 8.3 Use Limitation
 
-[Section 4.2 requirement 4](#4.2-r4) does not permit the CSP to use attributes collected and maintained in the identity proofing process for any purpose other than identity proofing, authentication, authorization, or attribute assertions, related fraud mitigation, or to comply with law or legal process unless the CSP provides clear notice and obtains consent from the subscriber for additional uses.
+[Section 4.2 requirement 4](#4.2-r4) requires CSPs to use measures to maintain the objectives of predictability (enabling reliable assumptions by individuals, owners, and operators about PII and its processing by an information system) and manageability (providing the capability for granular administration of PII, including alteration, deletion, and selective disclosure) commensurate with privacy risks that can arise from the processing of attributes for purposes other than identity proofing, authentication, authorization, or attribute assertions, related fraud mitigation, or to comply with law or legal process [NISTIR 8062]](#NISTIR 8062)
```
Comment (jimfenton) [inline sp800-63a/sec8_privacy.md]: Nits: assertions -> assertion
Need period at end

File: sp800-63a/sec8_privacy.md [hunk 2]
```diff
@@ -32,9 +32,11 @@ An effective notice will take into account user experience design standards and
...
+CSPs may have various business purposes for processing attributes, including providing non-identity services to subscribers. However, processing attributes for different purposes from the original collection purpose can create privacy risks when individuals are not expecting or comfortable with the additional processing. CSPs can determine appropriate measures commensurate with the privacy risk arising from the additional processing. For example, absent applicable law, regulation or policy, it may not be necessary to get consent when processing attributes to provide non-identity services requested by subscribers, although notices may help subscribers maintain reliable assumptions about the processing (predictability). Other processing of attributes may carry different privacy risks that call for obtaining consent or allowing subscribers more control over the use or disclosure of specific attributes (manageability). Subscriber consent needs to be meaningful; therefore, when CSPs do use consent measures, they cannot make acceptance by the subscriber of additional uses a condition of providing the identity service.
```
Comment (jimfenton) [inline sp800-63a/sec8_privacy.md]: Suggest changing “different purposes from the original collection purpose” to “other purposes than those specified at collection”.
Does the sentence on “appropriate measures” give CSPs an exemption from obtaining meaningful consent?

File: sp800-63b/sec4_aal.md [hunk 3]
```diff
@@ -155,7 +155,7 @@ The CSP shall comply with its respective records retention policies in accordanc
...
+If CSPs process attributes for purposes other than identity proofing, authentication, or attribute assertions (collectively “identity service”), related fraud mitigation, or to comply with law or legal process, CSPs SHALL implement measures to maintain predictability and manageability commensurate with the privacy risk arising from the additional processing. Measures MAY include providing clear notice, obtaining subscriber consent, or enabling selective use or disclosure of attributes. When CSPs use consent measures, CSPs SHALL NOT make consent for the additional processing a condition of the identity service.
```
Comment (jimfenton) [inline sp800-63b/sec4_aal.md:158]: Same comment as above.

File: sp800-63b/sec9_privacy.md [hunk 4]
```diff
@@ -19,7 +19,11 @@ CSPs should be able to reasonably justify any response they take to identified p
...
+
+CSPs may have various business purposes for processing attributes, including providing non-identity services to subscribers. However, processing attributes for different purposes from the original collection purpose can create privacy risks when individuals are not expecting or comfortable with the additional processing. CSPs can determine appropriate measures commensurate with the privacy risk arising from the additional processing. For example, absent applicable law, regulation or policy, it may not be necessary to get consent when processing attributes to provide non-identity services requested by subscribers, although notices may help subscribers maintain reliable assumptions about the processing (predictability). Other processing of attributes may carry different privacy risks that call for obtaining consent or allowing subscribers more control over the use or disclosure of specific attributes (manageability). Subscriber consent needs to be meaningful; therefore, as stated in [Section 4.4](#aal_privacy), when CSPs use consent measures, acceptance by the subscriber of additional uses SHALL NOT be a condition of providing authentication services.
```
Comment (jimfenton) [inline sp800-63b/sec9_privacy.md]: See comment above.

File: sp800-63c/sec5_federation.md [hunk 5]
```diff
@@ -134,7 +134,7 @@ Federation involves the transfer of personal attributes from a third party that
...
+If an IdP discloses information on subscriber activities at an RP to any party, or processes the subscriber’s information for any purpose other than identity proofing, authentication, or attribute assertions (collectively “identity service”), related fraud mitigation, to comply with law or legal process, or in the case of a specific user request, to transmit the information, the IdP SHALL implement measures to maintain predictability and manageability commensurate with the privacy risk arising from the additional processing. Measures MAY include providing clear notice, obtaining subscriber consent, or enabling selective use or disclosure of attributes. When an IdP uses consent measures, the IdP SHALL NOT make consent for the additional processing a condition of the identity service. The IdP SHOULD employ technical measures, such as the use of pairwise pseudonymous identifiers described in [Section 6.3](#ppi) or privacy-enhancing cryptographic protocols, to provide disassociability and discourage subscriber activity tracking and profiling.
```
Comment (jimfenton) [inline sp800-63c/sec5_federation.md:137]: See comment above.

**Review by jimfenton — CHANGES_REQUESTED** [review]

File: sp800-63a/sec8_privacy.md (hunk 2 above)
Comment (enadeau1) [reply sp800-63a/sec8_privacy.md]: I'm changing the other purposes sentence - yours sounds clearer. Thanks for that edit.

Re: consent, I think the language makes it clear that consent might be required by law; if not, then it's a risk-based decision. Yes?

**Review by paul-grassi — APPROVED** [review]

**Pull request merged**